README.md

```diff
@@ -283,15 +283,15 @@ var ErrVersion = errors.New("version mismatch")
 type Error struct {}
 ```
 
-An Error represents a runtime error, e.g. a `TypeError`, a `ReferenceError`, etc.
+An Error represents a runtime error, e.g., a `TypeError`, a `ReferenceError`, etc.
```

Comment:
suggestion: no change needed. Omitting the comma is fine, only considered more correct in **American** English